porcelain/protocol.go

```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/filecoin-project/go-filecoin/types"
 )
 
+// ProtocolParams TODO(rosa)
```

Comment:
ProtocolParams are a commonly inspected set of parameters for the storage and retrieval protocols that we make available for the user to inspect via the `go-filecoin protocol` command.